openfisca_us/entities.py

```diff
@@ -8,6 +8,16 @@
 
 from openfisca_core.entities import build_entity
 
+
+BenefitUnit = build_entity(
+    key="benefitunit",
+    plural="benefitunits",
+    label="The benefit unit",
+    doc="""
+    Description of a benefit unit
+    """,
+)
```

Comment:
Error is being caused by `build_entity` needing a `roles` argument, see here from openfisca-uk:
```python
roles=[
        {
            "key": "adult",
            "plural": "adults",
            "label": u"Adult",
            "doc": u"The adults in the household.",
        },
        {
            "key": "child",
            "plural": "children",
            "label": u"Child",
            "doc": u"Dependent children in the household.",
        },
    ],
```

Reply:
Does it need separate roles? I don't think there's a "head of household" concept for SPM units.